### PR TITLE
DOC: Fix typos in Modules/Core subdirectory

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S ./Modules/ThirdParty,./Documentation/ReleaseNotes -L ans,ba,childrens,developpement,dum,fiter,inout,ist,nd,ot,serie,te,unser ./Modules`

## PR Checklist
- [x] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [x] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [ ] Added test (or behavior not changed)
- [ ] Updated API documentation (or API not changed)
- [ ] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [ ] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [ ] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

Refer to the [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) for
further development details if necessary.

### CODE REVIEW

File: Modules/Core/Common/test/itkCommandObserverObjectTest.cxx [hunk 1]
```diff
@@ -303,7 +303,7 @@ testLambdaCommand()
     /*----- SECOND OBSERVER LAMBDA */
     o->AddObserver(itk::AnyEvent(), [&objRef, &name_of_class_cnt](const itk::EventObject & event) {
       ++name_of_class_cnt;
-      std::cout << "Ivocation # " << name_of_class_cnt << "\nObject: " << objRef.GetNameOfClass() << " Event: " << event
+      std::cout << "Invocation # " << name_of_class_cnt << "\nObject: " << objRef.GetNameOfClass() << " Event: " << event
                 << std::endl;
```
Comment (dzenanz) [inline Modules/Core/Common/test/itkCommandObserverObjectTest.cxx:307]: This change pushes this line over the max column limit defined by our style.
```suggestion
      std::cout << "Invocation # " << name_of_class_cnt << "\nObject: " << objRef.GetNameOfClass()
                << " Event: " << event << std::endl;
```

**Review by dzenanz — COMMENTED** [review]
Great! Only a minor linting error.

**Comment by luzpaz** [comment]
shall i squash and force push or do you want to take care of that?

**Comment by dzenanz** [comment]
It is preferable that you squash.

**Comment by luzpaz** [comment]
Done, [a16e0fa](https://github.com/InsightSoftwareConsortium/ITK/pull/3500/commits/a16e0faa02dcb8d58de061286e472c49898d0b0d)

**Review by dzenanz — APPROVED** [review]

**Review by N-Dekker — APPROVED** [review]
Double-checked 👍 Thanks @luzpaz !

**Pull request merged**